acme_unified.yml

```diff
@@ -0,0 +1,231 @@
+- mpfr=3.1.5=0
+- mpich=3.2=5
+- mpmath=1.0.0=py_0
+- msgpack-python=0.4.8=py27_0
+- nb_conda=2.2.1=py27_0
+- nb_conda_kernels=2.1.0=py27_0
+- nbconvert=5.3.1=py_1
+- nbformat=4.4.0=py27_0
+- nco=4.6.9=1
+- ncurses=5.9=10
+- netcdf-fortran=4.4.4=4
+- netcdf4=1.2.9=py27_1
+- notebook=5.2.1=py27_2
+- numpy=1.13.3=py27_blas_openblas_200
+- olefile=0.44=py27_0
+- openblas=0.2.19=2
+- openssl=1.0.2l=0
+- ossuuid=1.6.2=0
+- owslib=0.15.0=py_0
+- pandas=0.21.0=py27_0
+- pandoc=2.0.0.1=1
+- pandocfilters=1.4.1=py27_0
+- partd=0.3.8=py27_0
+- pathlib2=2.3.0=py27_0
+- pcre=8.39=0
+- pexpect=4.2.1=py27_0
+- pickleshare=0.7.4=py27_0
+- pillow=4.3.0=py27_1
+- pip=9.0.1=py27_0
+- proj4=4.9.3=5
+- prompt_toolkit=1.0.15=py27_0
+- psutil=5.4.0=py27_0
+- ptyprocess=0.5.2=py27_0
+- py=1.5.0=py_0
+- pycparser=2.18=py27_0
+- pyepsg=0.3.2=py27_0
+- pygments=2.2.0=py27_0
+- pyopenssl=17.2.0=py27_0
+- pyparsing=2.2.0=py27_0
+- pyproj=1.9.5.1=py27_0
+- pyqt=5.6.0=py27_4
+- pyshp=1.2.12=py_0
+- pysocks=1.6.7=py27_0
+- pytest=3.2.2=py27_1
+- python=2.7.14=1
+- python-dateutil=2.6.1=py27_0
+- pytz=2017.3=py_2
+- pyyaml=3.12=py27_1
+- pyzmq=16.0.2=py27_2
+- qt=5.6.2=6
+- qtconsole=4.3.1=py27_0
+- readline=6.2=0
+- requests=2.18.4=py27_1
+- scandir=1.6=py27_0
+- scipy=0.19.1=py27_blas_openblas_202
+- setuptools=36.6.0=py27_1
+- shapely=1.6.1=py27_1
+- simplegeneric=0.8.1=py27_0
+- singledispatch=3.4.0.3=py27_0
+- sip=4.18=py27_1
+- six=1.11.0=py27_1
+- sortedcontainers=1.5.7=py27_0
+- sqlite=3.13.0=1
+- ssl_match_hostname=3.5.0.1=py27_1
+- subprocess32=3.2.7=py27_0
+- sympy=1.1.1=py27_0
+- tblib=1.3.2=py27_0
+- terminado=0.6=py27_0
+- testpath=0.3.1=py27_0
+- tk=8.5.19=2
+- toolz=0.8.2=py_2
+- tornado=4.5.2=py27_0
+- traitlets=4.3.2=py27_0
+- udunits2=2.2.25=1
+- urllib3=1.22=py27_0
+- wcwidth=0.1.7=py27_0
+- webencodings=0.5=py27_0
+- wheel=0.30.0=py_1
+- widgetsnbextension=3.0.6=py27_0
+- x264=20131217=3
+- xarray=0.9.6=py27_0
+- xorg-libxau=1.0.8=3
+- xorg-libxdmcp=1.1.2=3
+- xz=5.2.3=0
+- yaml=0.1.6=0
+- zeromq=4.2.1=1
+- zict=0.1.3=py_0
+- zlib=1.2.11=0
+- cloog=0.18.0=0
+- gcc=4.8.5=7
+- isl=0.12.2=0
+- libgcc=7.2.0=h69d50b8_2
+- libgcc-ng=7.2.0=h7cc24e2_2
+- libgfortran=3.0.0=1
+- libstdcxx-ng=7.2.0=h7a57d05_2
+- progressbar=2.3=py27_0
+- pyevtk=1.0.0=py27_0
+- cdp=1.1.0=0
+- dv3d=2.12=py27hac3d9b5_0
+- output_viewer=1.2.2=py27_0
+- vcs=2.12=py27had75802_0
+- vcsaddons=2.12=py27h7706516_0
+- vtk-cdat=7.1.0.2.12=py27hd6333ce_0
+- acme-unified=1.1.2=h090690d_0
+- pip:
+  - acme-diags==1.0.0
+  - backports-abc==0.5
+  - backports.shutil-get-terminal-size==1.0.0
+  - backports.ssl-match-hostname==3.5.0.1
+  - cdat-info==0.0.0
+  - cibot==0.2
+  - ipython-genutils==0.2.0
+  - jupyter-client==5.1.0
+  - jupyter-console==5.2.0
+  - jupyter-core==4.4.0
+  - mv2==3.0.0
+  - nb-conda==2.2.1
+  - nb-conda-kernels==2.1.0
+  - output-viewer==1.2.2
+  - prompt-toolkit==1.0.15
+  - regrid2==3.0.0
+prefix: /export_2/shaheen2/anaconda2/envs/acme_unified_env-might-work-charles2
```

Comment:
Can we remove this before the PR gets merged?

Reply:
I'm guessing you're talking about the `prefix` (line 230) right? That might break something. In our other yml's, we keep this prefix and it works fine across many machines with different users. But I will change the prefix to `/export_2/shaheen2/anaconda2/envs/acme_unified_env`.

I also need to remove the build numbers as well.

Reply:
Hmm, I really don't understand how the prefix can be useful but if you feel like it's needed you can change it as you suggest.

Reply:
https://stackoverflow.com/questions/41274007/anaconda-export-environment-file
According to this post, the prefix gets ignored and can be removed.

Reply:
@xylar @zshaheen yes I would hope it's ignored.